packages/patternfly-4/react-core/src/components/Alert/AlertActionCloseButton.js

```diff
@@ -17,13 +17,13 @@ const propTypes = {
 };
 
 const defaultProps = {
+  'aria-label': '',
   className: '',
   onClose: () => undefined,
-  'aria-label': 'Close'
 };
 
-const AlertActionCloseButton = ({ className, onClose, 'aria-label': ariaLabel, ...props }) => (
-  <Button variant={ButtonVariant.plain} onClick={onClose} aria-label={ariaLabel} {...props}>
+const AlertActionCloseButton = ({ className, onClose, 'aria-label': ariaLabel, title, variantLabel, ...props }) => (
+  <Button variant={ButtonVariant.plain} onClick={onClose} aria-label={ariaLabel ? ariaLabel : `Close ${variantLabel} alert: ${title}`} {...props}>
```

Comment:
This wont work.  You need to either set the default to null or check for the empty string.

Reply:
I'm dumb. Amended the commit to reflect this.